tests/getScrollBehavior.test.js

```diff
@@ -0,0 +1,60 @@
+import getScrollBehavior from '../src/getScrollBehavior';
+
+const ScrollBehavior = getScrollBehavior();
+
+describe('ScrollBehavior', () => {
```

Comment:
Can we remove unnecessary describes?

Reply:
Can you do the same on the lib template?

Reply:
Removed describes and opened a MR in the package lib-template ✅